src/Utilities/sagas.js

```diff
@@ -64,9 +76,14 @@ function* listenForLoadSongsAllTime() {
   yield takeLatest('LOAD_SONGS_ALL_TIME', getSongsAllTime);
 }
 
+function* listenForPostPlaylist() {
+  yield takeLatest('POST_PLAYLIST', postPlaylistToProfile)
+}
+
 export default [
   listenForAuth,
   listenForLoadSongs,
   listenForLoadSongsShortTerm,
-  listenForLoadSongsAllTime
+  listenForLoadSongsAllTime,
+  listenForPostPlaylist,
 ];
```

Comment:
Well this is neat. I might bother you in the future if I try to use Sagas.

Reply:
I definitely prefer it! Slight learning curve but more readable off the bat and easy to test! ...which i still have to do ....